src/ParcelRegistry/Parcel/ValueObjects/VbrCaPaKey.cs

```diff
@@ -0,0 +1,20 @@
+namespace ParcelRegistry.Parcel
```

Comment:
why is there a new valueobject, I thought we moved it to higher level for reuse.